src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -307,6 +307,13 @@ public void setExampleSubmission(Boolean exampleSubmission) {
      */
     public abstract boolean isEmpty();
 
+    /**
+     * used to distinguish the type when used in collections (e.g. SearchResultPageDTO --> resultsOnPage)
```

Comment:
I don't get the `SearchResultPageDTO --> resultsOnPage` part 🤔 Do you intend to reference an attribute? Can you use `@see` for that?